meta-luneos/recipes-webos/configd-data/configd-data.bb

```diff
@@ -11,12 +11,20 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7ca
 inherit webos_ports_ose_repo
 
 PV = "1.0.0-2+git${SRCPV}"
-SRCREV = "8b41d8c3fd7f3e64fb9f18d4db25a223b42a191e"
+SRCREV = "790f21717864e9a55f8f6b159b9ded40d8d276de"
 
 SRC_URI = "${WEBOS_PORTS_GIT_REPO_COMPLETE}"
 S = "${WORKDIR}/git"
 
+PACKAGE_ARCH = "${MACHINE_ARCH}"
```

Comment:
Add configd->configd-data in meta-luneos/conf/layer.conf SIGGEN_EXCLUDE_SAFE_RECIPE_DEPS otherwise configd (and everything which depends on it) will became effectively MACHINE_ARCH as well.

Reply:
Isn't the goal of a config recipe to be MACHINE_ARCH? And then all the others RDEPEND on it?

Reply:
That's correct, but SIGGEN_EXCLUDE_SAFE_RECIPE_DEPS is needed to say that configd doesn't need to rebuild for every MACHINE because of configd-data runtime dependency - it marks the dependency between configd and configd-data to be ABI SAFE, similarly to SIGGEN_EXCLUDERECIPES_ABISAFE which just marks it ABI SAFE for every possible use of configd-data.

Reply:
@shr-project See https://github.com/webOS-ports/meta-webos-ports/pull/506/commits/3b86b29f4ae018d87b1ff9ce77f2ca1a68fff500